.circleci/config.yml

```diff
@@ -72,5 +72,5 @@ jobs:
               bin/ci/deploy-dockerhub.sh "circle_build_$CIRCLE_BUILD_NUM"
             fi
             if [ -n "${CIRCLE_TAG}"]; then
```

Comment:
I'm pretty sure you need a space between `"` and `]` at the end of this test, too.